### PR TITLE
Feature/sc 35723/re brand help center sub docs

### PR DESCRIPTION
**Pull request by NTerpo**

### CODE REVIEW

**Review by etienneburdet — CHANGES_REQUESTED** [review]
Overall looking good! 
The shadow is interupted by the sidebar: 
image
`.wy-nav-side { z-index: 100 }` seems to fix that nicely.

The language selector is still blue. 
```css
.rst-versions, 
.rst-versions .rst-current-version,
.rst-versions .rst-other-versions,
.rst-versions .rst-current-version .fa {
  background: white;
  color: black;
}

.rst-versions {
  border-top: 1px solid #DEE5EF;
}
```
Seems to fix it well.

It seems there is something that went differently in the JS build (a different node version or something), which has parts not minified and a few changes to the lock file. Seems to work though 🤷

File: source/_themes/ods-theme/static/js/theme.js [hunk 1]
```diff
@@ -1 +1,271 @@
...
\ No newline at end of file
+require=(function(){function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s}return e})()({"ods-theme":[function(require,module,exports){
+var jQuery = (typeof (window) != 'undefined') ? window.jQuery : require('jquery');
```
[the remark this answers is not in the document]
Comment (NTerpo) [reply source/_themes/ods-theme/static/js/theme.js:2]: I guess automatically generated.

File: source/_themes/dev/sass/_button.sass [hunk 2]
```diff
@@ -36,6 +36,12 @@ button
   line-height: normal
   -webkit-appearance: button
   *overflow: visible
+
+  padding: 8px
+  border-radius: 4px
```
[the remark this answers is not in the document]
Comment (NTerpo) [reply source/_themes/dev/sass/_button.sass:41]: Let's do it then, I'll update the landing page too

**Comment by NTerpo** [comment]
I've let the lang selector in blue : there were no figma for the old doc and all black & white looked a bit too sad. Otherwise I've made the fixes

**Review by etienneburdet — APPROVED** [review]

**Pull request merged**